tests/Fhi.HelseIdSelvbetjening.AcceptanceTests/AcceptanceTest.cs

```diff
@@ -102,6 +101,60 @@ public async Task UpdateClientKeysFromParameters()
             await host.StopAsync(TimeSpan.FromSeconds(10));
         }
 
+        [Test]
+        public async Task VerifyNewKeys_WithGeneratedKey_Succeeds()
+        {
+            var originalOutput = Console.Out;
+            var originalError = Console.Error;
+
+            await using var stringWriter = new StringWriter();
+            await using var errorWriter = new StringWriter();
+            Console.SetOut(stringWriter);
+            Console.SetError(errorWriter);
+
+            try
+            {
+                var testDirectory = Path.Combine(Environment.CurrentDirectory, "TestData");
+                var privateJwkFilePath = Path.Combine(testDirectory, "oldkey.json");
+                var privateJwkContent = await File.ReadAllTextAsync(privateJwkFilePath);
+
+                var verifyArgs = new[]
+                {
+                    "verifynewkeys",
+                    "--clientId", 
+                    "88d474a8-07df-4dc4-abb0-6b759c2b99ec",
+                    "--privateJwk",
+                    privateJwkContent
+                };
+                 
+                stringWriter.GetStringBuilder().Clear();
+                errorWriter.GetStringBuilder().Clear();
+
+                var exitCode = await Program.Main(verifyArgs);
```

Comment:
The exitcode is not returned from the Main routine as anything but 0, since it is being set by Environment.Exitcode and not returned from the Main routine.
![image](https://github.com/user-attachments/assets/e2bb5d34-56ac-4fcd-933e-b61b33987adf)